package.json

```diff
@@ -26,17 +26,18 @@
   },
   "main": "index.js",
   "dependencies": {
+    "@bcgov/design-tokens": "^3.0.1",
     "@wordpress/icons": "^10.3.0"
   },
   "devDependencies": {
     "@bcgov/bc-sans": "^2.1.0",
     "@bcgov/wordpress-eslint": "git+https://apps.itsm.gov.bc.ca/bitbucket/scm/wp/bcgov-wordpress-eslint.git#semver:1.x",
     "@bcgov/wordpress-stylelintrc": "git+https://apps.itsm.gov.bc.ca/bitbucket/scm/wp/bcgov-wordpress-stylelintrc.git#semver:1.x",
-    "@wordpress/scripts": "^28.3.0"
+    "@wordpress/scripts": "^28.6.0"
   },
   "scripts": {
-    "build": "wp-scripts build src/scripts/public/font.js --output-path=dist",
-    "build:production": "npm ci && npm run fix:js && npm run lint:js && npm run fix:css && npm run lint:css && npm run lint:md:docs && npm run test:unit && npm run build",
+    "build": "wp-scripts build --output-path=dist",
+    "build:production": "npm ci && npm run fix:js && npm run lint:js && npm run fix:css && npm run lint:css && npm run test:unit && npm run build",
```

Comment:
most package.json i see have the `lint:md:docs` removed. It's handy, I think, for making markdown documents consistent with our best practices for formatting, header order etc.

I wonder what issues having it here causes?